File: haushaltsbuch/src/test/java/org/github/haschi/haushaltsbuch/DieWelt.java
package org.github.haschi.haushaltsbuch;

import org.assertj.core.api.ThrowableAssert;
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Aggregatkennung;

@SuppressWarnings("WeakerAccess")
public class DieWelt
{
    public Aggregatkennung aktuelleInventur;
    public ThrowableAssert.ThrowingCallable intention;
    public Aggregatkennung aktuellesHaushaltsbuch;
}

File: haushaltsbuch/src/test/kotlin/org/github/haschi/haushaltsbuch/DieWelt.kt
package org.github.haschi.haushaltsbuch

import org.github.haschi.haushaltsbuch.modell.core.values.Aggregatkennung

class DieWelt {
    var aktuelleInventur: Aggregatkennung? = null
    var intention: (() -> Unit)? = null
    var aktuellesHaushaltsbuch: Aggregatkennung? = null
}
